vpr/src/place/cut_spreader.h

```diff
@@ -0,0 +1,73 @@
+#ifndef VPR_SRC_PLACE_LEGALIZER_H_
```

Comment:
File needs more commenting.  Each data member should be commented to indicate what it is for.